### PR TITLE
spans only schemas

### PR DESCRIPTION
**Pull request by dbanda**
New schema for the spans only pipeline.

### CODE REVIEW

File: examples/snuba-spans/1/basic_span.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "parent_span_id": "deadbeefdeadbeef",
+    "segment_id": "deadbeefdeadbeef",
+    "is_segment": false,
+    "start_timestamp_ms": 1691105878720,
+    "duration_ms": 1000,
+    "exclusive_time_ms": 1000,
+    "tags": [
+      ["tag1", "value1"],
+      ["tag2", "value2"]
+    ],
```
Comment (nikhars) [inline examples/snuba-spans/1/basic_span.json:17]: This should be the same format as `sentry_tags` below i.e. a map of key:value pairs. The snuba consumer should change it to make it an array of keys and values when writing it to clickhouse.

File: examples/snuba-spans/1/basic_span.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+[
+  2,
```
Comment (nikhars) [inline examples/snuba-spans/1/basic_span.json:2]: This version is setup when an event is sent through the eventstream. Not sure whether spans would be sent via the eventstream or some other interface.

File: examples/snuba-spans/1/basic_span.json (hunk 1 above)
Comment (dbanda) [reply examples/snuba-spans/1/basic_span.json:17]: I copied this from the way its done in transactions. Should I still switch it to a map?

File: examples/snuba-spans/1/basic_span.json (hunk 2 above)
Comment (dbanda) [reply examples/snuba-spans/1/basic_span.json]: removed

File: examples/snuba-spans/1/basic_span.json (hunk 1 above)
Comment (dbanda) [reply examples/snuba-spans/1/basic_span.json]: fixed

Comment (nikhars) [reply examples/snuba-spans/1/basic_span.json]: I wasn't aware that transactions sends tags as arrays of keys and values. Maybe @phacops can provide better guidance on what to expect here.

**Review by nikhars — APPROVED** [review]

**Pull request merged**